### DISCUSSION TITLE
Exported object names not minified since 5.79

### DISCUSSION THREAD

**Discussion by vjhebbar** [webpack/webpack#17988] [Q&A]
Hello,

Since I updated to webpack 5.79+ in my project, bundle sizes have increased because some exported object names are not minified.

It turns out that this is due to those objects being destructured in modules where they are imported. 
Refactoring the code to remove the object destructuring minimizes the name again. 

Prior to 5.79, the names were minified regardless of the use of destructuring. 

Is this behavior aligned with #16941?

If this is not expected behavior, I'll share my configs and minimum reproducible code.

Looking forward to your thoughts and suggestions.

The latest version where the issue is reproducible is 5.90.

**Comment by alexander-akait** [comment]
Please provide the minimum reproducible test repo, thank you

**Comment by vjhebbar** [comment]
@alexander-akait Here's a repo illustrating the issue. https://github.com/vjhebbar/webpack579test
Note that this is using dev mode. I couldn't repro with production mode on this simple configuration.

My production application is far more complex where I can reproduce the issue in production mode. I'm working on simplifying this production configuration to be reproducible as well.

For now, I'm fairly confident that the differences in the output shown in that repo in dev mode are the same as problem in the production mode and are caused by the same underlying issue.

Let me know if you need any other information.

**Reply by alexander-akait** [reply]
I see, looks like regression, let's open an issue, if you have a time - PR welcome

**Reply by vjhebbar** [reply]
Thanks @alexander-akait!
FYI - @vankop
